### DISCUSSION TITLE
NextJS 13.5 AVIF Images Not Working

### DISCUSSION THREAD

**Discussion by SaavanNanavati** [vercel/next.js#55668] [Help]
### Summary

I can't import static AVIF image files in NextJS 13.5 anymore. I get the following:

```Error: Image import ... is not a valid image file. The image may be corrupted or an unsupported format.```

Anyone else running into this?

It works fine on NextJS 13.4. Not having this issue with WEBP on NextJS 13.5.

**Comment by metawise** [comment]
Mine is working but "w" parameter not making changes. Size of the image remains the same no matter what I set it.

**Comment by ivanskodje** [comment]
Sharing this here, since I had the exact same problem. 

First i installed sharp: `yarn add sharp`
Secondly, I updated nextConfig to include avif image formats:

```mjs
const nextConfig = {
  output: "standalone",
  images: {
    formats: ["image/avif", "image/webp"],
  },
};
```
